src/azul/__init__.py

```diff
@@ -191,6 +191,7 @@ def num_dss_workers(self) -> int:
     @property
     def external_lambda_role_assumptors(self) -> MutableMapping[str, List[str]]:
         try:
+            # This env variable should only be set for HCA deployments
```

Comment:
I wouldn't go that far. Currently only the HCA deployments have this set. But that could change in the future in which case this comment would become incorrect.